src/structs/field-shorthand.md

```diff
@@ -21,3 +21,13 @@ fn main() {
     println!("{peter:?}");
 }
 ```
+
+<details>
+ Key Points: 
+    
+* This slide illustrates short-hand syntax, by initiating the fields in a struct with similar-named variables. 
```

Comment:
Can you make this text more direct? Please try to elaborate on the text in the slide instead of reformulating it.